README.md

```diff
@@ -47,8 +45,8 @@ The default SSH root password is `gentoo-root`. This is the list of accepted env
 * Install `sys-boot/syslinux` as it contains the necessary files:
 
 ```
-$ sudo cp /usr/share/syslinux/ldlinux.c32 /pxe/
-$ sudo cp /usr/share/syslinux/pxelinux.0 /pxe/
+# cp /usr/share/syslinux/ldlinux.c32 /pxe/
+# cp /usr/share/syslinux/pxelinux.0 /pxe/
```

Comment:
This change is on the assumption that current user already has write permission to /pxe (trying really hard to avoid need for sudo here)

Reply:
Then maybe add a test and failure + error message to check this assumption?

Reply:
Since it is [documentation](https://github.com/numberly/gentoo-pxe-builder#pxe-server-basic-setup) rather than script I was hoping that most would understand which permissions they need?

But maybe add a line with something like:
"Change permissions on `/pxe` so that you can write without being root `su -c "chown $(whoami) /pxe"`, Using the right permissions in Linux is an essential skill to have"
But I'm not at all sure what the best approach here is?